.github/workflows/test.yml

```diff
@@ -30,7 +30,7 @@ jobs:
   pwa-kit:
     strategy:
       matrix:
-        node: [14]
+        node: [14, 16]
```

Comment:
Can we also trigger a Node 16 build on Windows by adding 16 to the windows job matrix, please?
https://github.com/SalesforceCommerceCloud/pwa-kit/blob/8b235fa54f711eb80abd1f3987d6e1f45720fd79/.github/workflows/test.yml#L119-L124

Reply:
Done! Waiting to see if the tests pass 🤞🏼

Reply:
Sorry, @raiyaj since Node 16 is the active Node version, Do you mind also updating the generated builds to use Node 16?

https://github.com/SalesforceCommerceCloud/pwa-kit/blob/ea563c5cbd5a51f448e41d263819c9e33d40e1f8/.github/workflows/test.yml#L160

https://github.com/SalesforceCommerceCloud/pwa-kit/blob/ea563c5cbd5a51f448e41d263819c9e33d40e1f8/.github/workflows/test.yml#L250